css/store.css

```diff
@@ -0,0 +1,289 @@
+header {
+  height: 100px;
+  display: flex;
+  position: relative;
```

Comment:
"I used position: relative on the header because I want to reposition the navigation when applying media queries for mobile devices. On mobile, the navigation (header nav) is set to position: absolute, allowing me to control its position using the top, bottom, left, and right properties. By setting the parent (header) to position: relative, I ensure that the absolutely positioned navigation is placed relative to the header, making it easier to position the nav exactly where needed for mobile layouts."

Reply:
Makes sense. Nice one